irida_staramr_results/cli.py

```diff
@@ -49,12 +57,63 @@ def _validate_args(args):
     if args.output.endswith(".xlsx"):
         args.output = args.output[:-len(".xlsx")]
 
+    date = _validate_date(args.fromDate, args.toDate)
+
     return {'username': args.username,
             'password': args.password,
             'config': args.config,
             'project': args.project,
             'output': args.output,
-            'append': args.append}
+            'append': args.append,
+            'fromDate': date["fromDate"],
+            'toDate': date["toDate"]}
+
+
+def _validate_date(from_date, to_date):
+    """
+    Sets up FROM and TO date values in unix timestamp.
+    :param from_date:
+    :param to_date:
+    :return:
+    """
+
+    if from_date is None:
+        from_date = 0
+    else:
+        from_date = local_to_timestamp(from_date)
+
+    if to_date is None:
+        to_date = time.time() * 1000
+    else:
+        to_date = local_to_timestamp(to_date)
+
+    if (to_date > time.time() * 1000) or (from_date > time.time() * 1000):
+        logging.error("DateError: --fromDate and --toDate cannot be in the future.")
+        sys.exit(1)
+
+    if from_date > to_date:
+        logging.error("DateError: --fromDate must be earlier than --toDate.")
+        sys.exit(1)
+
+    # Add 24 hours (86400000 milliseconds) to include to_date's full day.
+    to_date = to_date + 86400000
+
+
+    return {"fromDate": from_date, "toDate": to_date}
+
+
+def local_to_timestamp(target_date):
```

Comment:
This function should be private (starts with `_`)

Reply:
done